server/uploader.js

```diff
@@ -230,6 +250,26 @@ uploader.bedSort = function(prefix, bedFile) {
       }
       fs.closeSync(fd);
     }
+
+    var fd = fs.openSync(uploadPath + bedFile + '.finish', 'w');
+    fs.writeSync(fd, 'finished');
+    fs.closeSync(fd);
     console.log('bed chromosome data finish.');
   });
 };
+
+/**
+ * Checks whether the file uploaded.
+ * @param {{
+ *  fileName: string
+ * }} query Parameters for file checking.
+ * @param {string} prefix Path to the folder.
+ * @return {boolean} File exists or not
+ */
+uploader.checkFinish = function(query, prefix) {
+  var isFinish = fs.existsSync(prefix + query.fileName + '.finish');
+  if (isFinish) {
+    fs.unlinkSync(prefix + query.fileName + '.finish');
```

Comment:
I think you may instead create a "pending" file at the beginning of the processing and remove that file after processing is complete.
isFinish = notExist(pendingFile)